src/OpenKNX/Console.cpp

```diff
@@ -2,7 +2,7 @@
 #include "OpenKNX/Facade.h"
 #include "OpenKNX/Flash/Driver.h"
 
-#ifdef ARDUINO_ARCH_RP2040
+#if defined(ARDUINO_ARCH_RP2040) || defined(ARDUINO_ARCH_ESP32)
```

Comment:
`if defined(ARDUINO_ARCH_RP2040) || defined(ARDUINO_ARCH_ESP32)` is used 12 times for FS-specific implementation.

Let's check definition of own symbol for file-system-feature. @mgeramb @traxanos

Reply:
Done